app/main.py

```diff
@@ -1 +1,44 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str, appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
+
+
+class Cat(Animal):
+    def __init__(self, name: str,
+                 is_hungry: bool = True) -> None:
+        super().__init__(name=name, appetite=3, is_hungry=is_hungry)
+
+    @staticmethod
```

Comment:
The `@staticmethod` decorator is correctly used here, but it might be more appropriate to use an instance method if `catch_mouse` is intended to be called on a specific `Cat` instance. Consider whether this method should interact with instance attributes.